.gitignore

```diff
@@ -33,4 +33,7 @@ node_modules/
 storybook-dist
 
 # vscode
-.vscode
\ No newline at end of file
+.vscode
+
+# ignore js locales 
+packages/L10n/src/locales-js/
```

Comment:
I think this should be added to the local version of `.gitignore` in `/packages/l10n`